modules/javafx.media/src/main/native/gstreamer/plugins/mfwrapper/mfwrapper.h

```diff
@@ -103,6 +104,8 @@ struct _GstMFWrapper
     guint defaultStride;
     guint pixel_num;
     guint pixel_den;
+
+    gboolean set_caps;
```

Comment:
may be rename as `is_set_caps` similar to the other boolean variables of the class.